### PR TITLE
Updated FAQ page and contribution links

### PR DESCRIPTION
**Pull request by TrechNex**

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -103,62 +104,56 @@ Just like many other free software projects we need to host the collaboration to
...
+Our [Open Collective](https://opencollective.com/glimpse) and [Patreon](https://www.patreon.com/glimpse) backers also get regular updates about the project. Our more in-depth daily discussions can be viewed [on Matrix]([#glimpse:matrix.org](https://matrix.to/#/#glimpse:matrix.org)).
 
 [Return to top](#contents)
 
 ## How do I contribute to this project? {#how-do-i-contribute-to-this-project}
-You can spread the word when we share news on [the fediverse](https://bobadon.co.uk/@glimpse) and [Twitter](https://twitter.com/glimpse_editor). You can also donate to our [Patreon](https://www.patreon.com/glimpse) campaign.
-
-Join us on [#glimpse:matrix.org](https://matrix.to/#/#glimpse:matrix.org) if you want to participate in the project's decision-making process. You can also raise bugs and work on tasks via our [Github Issues](https://github.com/glimpse-editor/Glimpse/issues) page.
+All of our contribution links are available on our [Contribute](../contribute/) page.
```
Comment (BrainBlasted) [inline content/about.md]: This can just be `/contribute/`.

File: content/about.md [hunk 2]
```diff
@@ -103,62 +104,56 @@ Just like many other free software projects we need to host the collaboration to
...
 
-Please ensure that you read our [code of conduct](../code-of-conduct/) before you join our project's community. Regardless of your past experience and the usefulness of your contributions you are expected to comply with it.
+Please ensure that you read our [code of conduct](../code-of-conduct/) before you join our community. Regardless of your past experience and the usefulness of your contributions you are expected to comply with it.
```
Comment (BrainBlasted) [inline content/about.md]: Also `/code-of-conduct/`

File: content/about.md [hunk 3]
```diff
@@ -103,62 +104,56 @@ Just like many other free software projects we need to host the collaboration to
...
 
 ## Which operating systems do you support? {#which-operating-systems-do-you-support}
-*This answer is provisional and subject to change*
-
-We intend to support recent desktop versions of Windows, Mac and Linux.
+We support systems running Windows 7 or newer, and modern variants of GNU/Linux.
 
-Linux users will be able to install Glimpse as a [Flatpak](https://flatpak.org/) or an [AppImage](https://appimage.org/).
-
-A source tarball will be provided for package maintainers of Linux, BSD and Solaris distributions.
```
Comment (BrainBlasted) [inline content/about.md]: Why remove this section?

Comment (TrechNex) [reply content/about.md]: The way we choose to package the project may change over time. I also don't think this question needs that level of details. Open to suggestions on this though.

**Pull request merged**